### PR TITLE
feat: support to load the image cache into docker

### PR DESCRIPTION
**Pull request by LinuxSuRen**
The e2e testing need the local image. See also https://github.com/LinuxSuRen/halo/pull/1

### CODE REVIEW

File: halo-next-docker-build/action.yaml [hunk 1]
```diff
@@ -18,6 +18,10 @@ inputs:
     description: Should push the docker image or not.
     required: false
     default: "false"
+  load:
+    description: Should load the image image into docker or not.
```
Comment (guqing) [inline halo-next-docker-build/action.yaml]:
```suggestion
    description: Should load the image into docker or not.
```

Comment (LinuxSuRen) [reply halo-next-docker-build/action.yaml]: Good catch!

**Review by JohnNiang — APPROVED** [review]
/approve

**Review by guqing — COMMENTED** [review]
/lgtm

**Pull request merged**